pyproject.toml

```diff
@@ -85,11 +87,11 @@ pydicom = { version = ">=2.0.0", optional = true } # groups = ["user", "dev", "d
 shapely = { version = ">=1.7.0", optional = true }       # groups = ["user", "dev"]
 scikit-image = { version = ">=0.18.1", optional = true } # groups = ["user", "dev"]
 
-# Streamlit's pinning is due to its current beta nature and the fact that some
-# private APIs are utilised.
-protobuf = { version = "<3.19.0", optional = true }    # groups = ["user", "dev"]
-streamlit = { version = "1.4.0", optional = true }     # groups = ["user", "dev"]
-streamlit-ace = { version = "0.1.0", optional = true } # groups = ["user", "dev"]
+# Utilising the pymedphys fork of pylibjpeg-libjpeg to fix the following issue:
+# https://github.com/pymedphys/pymedphys/issues/1556#issuecomment-1152821285
+pymedphys-pylibjpeg-libjpeg = { version = "1.3.1", optional = true } # groups = ["user", "dev"]
```

Comment:
is this likely to  cause version clash problems with other software that uses PyMedPhys as a library/package?

Reply:
The hope is that this will be removed once the proper package undergoes its own update. I've already flagged a request to get a release out, so hopefully we'll be able to do that before a new PyMedPhys release, or maybe even before merging this PR...

https://github.com/pydicom/pylibjpeg-libjpeg/issues/38